src/server/cluster/cluster_family.h

```diff
@@ -51,17 +53,48 @@ class ClusterFamily {
   void DflyMigrate(CmdArgList args, ConnectionContext* cntx);
 
   void MigrationConf(CmdArgList args, ConnectionContext* cntx);
+  void Flow(CmdArgList args, ConnectionContext* cntx);
```

Comment:
I think that it is confusing and not clear which function / data structurs are used in the source/target node. 
So please add comments and also it will be easier if they we be next to each other in the class.